tests/Doctrine/Tests/DBAL/Types/DateImmutableTypeTest.php

```diff
@@ -17,7 +17,7 @@ class DateImmutableTypeTest extends \PHPUnit\Framework\TestCase
     private $platform;
 
     /**
-     * @var DateImmutableType
+     * @var Type
```

Comment:
It would make more sense to instantiate the object via `new` instead of the factory and therefore having to loosen the type. Otherwise, we're testing the factory and the type.